tron/config/schema.py

```diff
@@ -109,6 +109,9 @@ def config_object_factory(name, required=None, optional=None):
 
 ConfigMesos = config_object_factory(
     name='ConfigMesos',
+    required=[
+        'master_address',
```

Comment:
Does this mean you will have to add `mesos_address` to all our master files when we upgrade?

Reply:
Yes. I will update the master files later.